docs/gettingStarted/ANGULAR.md

```diff
@@ -1 +1,242 @@
+{
+    "projects": [],
+    "styleguides": [
+        "5cd486b18a64c1414be004fb"
+    ],
+    "components": [
+        {
+            "path": "src/components/Button/Button.js",
+            "zeplinNames": [
+                "Controls / Button / Primary"
+            ]
+        }
+    ]
+}
+```
+
+Notice that in the screenshot above, we have two more states of the same button. It's possible connect a component in our codebase to multiple components in Zeplin. Here's how that would look like:
+
+```json
+…
+            "zeplinNames": [
+                "Controls / Button / Primary",
+                "Controls / Button / Primary, Hover",
+                "Controls / Button / Primary, Pressed"
+            ]
+…
+```
+
+If you're using the Visual Studio Code extension, you can simply click “Connect to Zeplin component” and search for a component in Zeplin, directly within Visual Studio Code.
+
+**Congratulations, we just connected our first component!** 🎉
+
+Next up, we'll install and use Zeplin's CLI tool so that these connected components are visible in Zeplin to our team.
+
+## 2. Install Zeplin CLI
+
+Zeplin CLI runs in your terminal and communicates the configuration file with Zeplin.
+
+Let's start by installing it. Zeplin CLI runs on Node.js, if you don't have it installed already, see [Node.js website](https://nodejs.org/en/).
+
+To install Zeplin CLI from npm, run the following command on your Terminal/Command Prompt:
+
+```sh
+npm install -g @zeplin/cli
+```
+
+Alternatively, if you're using npm in your project, you can add `@zeplin/cli` as a `devDependency`.
+
+## 3. Install CLI Angular plugin
+
+Zeplin CLI uses plugins to generate documentation, snippets and links from components—check out our [list of plugins](README.md#Plugins).
+
+Since we're using Angular, we'll install the official Angular plugin from npm, by running the following command:
+
+```sh
+npm install -g @zeplin/cli-connect-angular-plugin
+```
+
+Now, we'll update our configuration file to use the plugin. We can do that by adding it under the `plugins` list, like so:
+
+```json
+{
+    "plugins": [
+        {
+            "name": "@zeplin/cli-connect-angular-plugin"
+        }
+    ],
+    "projects": [],
+    "styleguides": [
+        "5cd486b18a64c1414be004fb"
+    ],
+    "components": [
+        {
+            "path": "src/components/Button/Button.ts",
+            "zeplinNames": [
+                "Controls / Button / Primary"
+            ]
+        }
+    ]
+}
+```
+
+Next up, we'll run the CLI tool!
+
+## 4. Run Zeplin CLI
+
+It's time! Let's **run the CLI tool and see Connected Components in action** within Zeplin. 🎉
+
+Run the following command—if it's your first time, you'll need to login to Zeplin first:
+
+```sh
+zeplin connect
+```
+
+Now head back to Zeplin and click on one of the components you connected. You should see an output similar to this:
+
+<img src="../../img/zeplinConnectedComponent-angular.png" alt="Connected component in Zeplin" width="600" />
```

Comment:
File is missing for now. I've created an example Angular component looking like React example, will how it is looking like asap.